test/fixed_stack_test.cpp

```diff
@@ -45,6 +45,13 @@ TEST(FixedStack, MaxSize)
         FixedStack<int, 3> v1{};
         EXPECT_EQ(3, v1.max_size());
     }
+
+    {
+        static_assert(FixedStack<int, 3>::static_max_size() == 3);
+        EXPECT_EQ(3, (FixedStack<int, 3>::static_max_size()));
```

Comment:
why the runtime check right after the same compile time check? is this a convention?

Reply:
Coverage testing tools are runtime based and don't work with constexpr so I have both versions for coverage metrics.